deploy/pivot_role/pivotRole.yaml

```diff
@@ -40,48 +40,25 @@ Resources:
             Condition:
               StringEquals:
                 'sts:ExternalId': !Ref ExternalId
+              StringLike:
+                'aws:PrincipalArn': [
+                  !Sub "arn:aws:iam::${AWS::AccountId}:role/*graphql-role",
```

Comment:
The Account Id here should be referencing the parameter Account Id and not the AWS::AccountId where the stack is being deployed. I think it should be 

```
                  !Sub "arn:aws:iam::${AccountId}:role/*graphql-role",
                  !Sub "arn:aws:iam::${AccountId}:role/*awsworker-role",
                  !Sub "arn:aws:iam::${AccountId}:role/*ecs-tasks-role"
```

so it is referencing the central infrastructure account